### DISCUSSION TITLE
Arbitrary values don't work when there are line breaks in the classname

### DISCUSSION THREAD

**Discussion by tsnobip** [tailwindlabs/tailwindcss#7137] [Help] [answered]
**What version of Tailwind CSS are you using?**

v3.0.15

**What build tool (or framework if it abstracts the build tool) are you using?**

vite 2.7.12, postcss-cli 8.4.5

**What version of Node.js are you using?**

v16.13.0

**What browser are you using?**

Firefox, Chrome

**What operating system are you using?**

macOS 11.6.2

**Reproduction URL**

https://github.com/tsnobip/tw-line-break-repro

**Describe your issue**

If you add a line break after an arbitrary value, the compiler doesn't seem to create the class, for example:
```
className="h-[40vmin]\n pointer-events-none"
```
Here `h-[40vmin]` won't get applied.

**Comment by adamwathan** [comment, marked as answer]
Hey! Not sure what you are trying to do here but your code doesn't actually introduce a line break, it just adds the `\` and `n` characters to the class name, you can see that right here in dev tools:

![image](https://user-images.githubusercontent.com/4323180/150204826-ad3a7b69-bc8b-4865-9cee-d954caadd9c5.png)

So the class name in your code is `h-[40vmin]\n` with literal backslash and `n` characters, not an actual newline character. When the browser looks at that list of classes, it sees `h-[40vmin]\n` which would match a CSS rule with this selector:

```css
.h-\[40vmin\]\\n {
  ...
}
```

What are you actually trying to do?

**Reply by tsnobip** [reply]
hmm yeah, I guess the issue comes from the tool I'm using (rescript) compiles 
```
`h-[40vmin]
  p-3`
```
to 
```
"h-[40vmin]\n  p-3" 
```
and that unfortunately react compiles it directly to the same classname without escaping the line break.

I guess the issue is either in rescript or react but in tailwind then!

Thanks a lot for your help!
